### PR TITLE
feat: Fund11 prod dates | NPG-000

### PR DESCRIPTION
**Pull request by kukkok3**
# Description

Adding fund 11 dates
The idea scale importer sql file will be updated in another ticket
Fixes https://github.com/input-output-hk/catalyst-core/issues/639

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: _if applicable_

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

**List of new dependencies**: _if applicable_

Provide a list of newly added dependencies in this PR, with the description of what are they doing and why do we need them.

- Crate A: description
- Crate B: description

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/event-db/stage_data/prod/00002_fund11_params.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        "nr_allocations": [30, 80],
+        "campaign_group_id": 63,
+        "questions": {
+            "You are reviewing the positive IMPACT this project will have on the Cardano Ecosystem.\nHas this project clearly demonstrated in all aspects of the proposal that it will have a positive impact on the Cardano Ecosystem?": "Impact / Alignment",
+            "You are reviewing the FEASIBILITY of this project.\nIs this project feasible based on the proposal submitted? Does the plan and associated budget and milestones look achievable? Does the team have the skills, experience, capability and capacity to complete the project successfully?": "Feasibility",
+            "You are reviewing the VALUE FOR MONEY this represents for the Treasury and the Community\nIs the funding amount requested for this project reasonable and does it provide good Value for Money to the Treasury?": "Auditability"
+        },
+        "stage_ids": [4590, 4596, 4602, 4608, 4614, 4620, 4626, 4632, 4638, 4644, 4650, 4656, 4662, 4591, 4597, 4603, 4609, 4615, 4621, 4627, 4633, 4639, 4645, 4651, 4657, 4663, 4592, 4598, 4604, 4610, 4616, 4622, 4628, 4634, 4640, 4646, 4652, 4658, 4664],
```
Comment (coire1) [inline src/event-db/stage_data/prod/00002_fund11_params.sql:16]: F11 stage ids: `4705, 4706, 4707, 4708, 4709, 4711, 4712, 4713, 4714, 4715, 4717, 4718, 4719, 4720, 4721, 4723, 4724, 4725, 4726, 4727, 4729, 4730, 4731, 4732, 4733, 4735, 4736, 4737, 4738, 4739, 4741, 4742, 4743, 4744, 4745`

File: src/event-db/stage_data/prod/00002_fund11_params.sql [hunk 2]
```diff
@@ -0,0 +1,61 @@
+-- Define F11 IdeaScale parameters.
+INSERT INTO config (id, id2, id3, value) VALUES (
+    'ideascale',
+    '11',
+    '',
+    '{
+        "group_id": 31051,
+        "review_stage_ids": [143, 145],
+        "nr_allocations": [30, 80],
+        "campaign_group_id": 63,
```
Comment (coire1) [inline src/event-db/stage_data/prod/00002_fund11_params.sql:10]: campaign_group_id: `94`

File: src/event-db/stage_data/prod/00002_fund11_params.sql [hunk 3]
```diff
@@ -0,0 +1,61 @@
+-- Define F11 IdeaScale parameters.
+INSERT INTO config (id, id2, id3, value) VALUES (
+    'ideascale',
+    '11',
+    '',
+    '{
+        "group_id": 31051,
```
Comment (coire1) [inline src/event-db/stage_data/prod/00002_fund11_params.sql:7]: group_id: `38124`

**Review by coire1 — CHANGES_REQUESTED** [review]

**Comment by kukkok3** [comment]
@coire1 the ideascale changes will be part of another PR. @saibatizoku is working on that. This PR is only to get the continuous snapshot going

**Review by coire1 — APPROVED** [review]
LGTM - Ideascale changes will be part of another PR

**Pull request merged**